flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/delegation/PlannerBase.scala

```diff
@@ -268,6 +268,50 @@ abstract class PlannerBase(
           getFlinkContext.getClassLoader
         )
 
+      case ctasOperation: CreateTableASOperation =>
```

Comment:
can we dedup more logic? in the end RTAS and CTAS are almost the same. They only differ in constructing one `getCreateTableOperation`?